integration/v7/push/manifest_diff_test.go

```diff
@@ -53,4 +53,43 @@ var _ = Describe("displaying manifest differences between pushes", func() {
 			})
 		})
 	})
+	FWhen("there is an app that has been pushed and we push it again with a different manifest", func() {
```

Comment:
@nouseforaname, can you please remove the focus?

Reply:
Done, thanks for spotting this.